docs/development/development.md

```diff
@@ -65,6 +65,12 @@ cargo version
     > Note: To test a specific component, use the `-p` parameter along with the [workspace member](https://github.com/project-akri/akri/blob/main/Cargo.toml). For example, to only test the Agent, run `cargo test -p agent`
 
 ## Running locally
+Before running Akri agent or controller locally, please ensure Akri configuration and instance CRDs are created in cluster, otherwise use the below command to create.
+```sh
+    kubectl apply -f ~/akri/deployment/helm/crds/akri-configuration-crd.yaml
```

Comment:
This path assumes that the `akri` repo is right under the home directory. I would change this to just `akri/deployment...` as in other parts of the development docs. 

Also some nits:
- can we change "created in" to "applied to" (and "to create" to "to apply them") to match the cli command?
- change to "please ensure the Akri..."

Reply:
thanks, I will comply